lib/DropdownSwitcher/index.js

```diff
@@ -51,9 +52,9 @@ class DropdownSwitcher extends Component {
   dropdown() {
     return (
       <Dropdown className="dropdown-switcher" id={this.props.id}>
-        <div bsRole="toggle">
+        <CustomToggle bsRole="toggle">
```

Comment:
What's the story with this new `CustomToggle`?

Reply:
Ah its a work around for a warning. Having bsRole on a div threw up a warning, the whole new dom attributes thing complained about it all not being lower case, but bootstrap needs it.